include/weak_forms/solution_storage.h

```diff
@@ -37,6 +37,73 @@ WEAK_FORMS_NAMESPACE_OPEN
 
 namespace WeakForms
 {
+  /**
+   * @brief An class that allows one to store solution vectors that may be referenced by the weak forms during assembly.
```

Comment:
> An class that allows one to store solution vectors that may be referenced by the weak forms during assembly.

A class that allows one to store solution vectors that may be referenced by the weak forms during assembly.

Reply:
Thanks! I've incorporated this into a patch that I'll apply on a future branch.